From 6765fd677168df46dbed3cb4c32b9104ce2d3e83 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Thu, 29 Dec 2011 20:52:18 -0500
Subject: [PATCH 079/183] tipc: rename struct bclink to struct tipc_bclink
Git-commit: 6765fd677168df46dbed3cb4c32b9104ce2d3e83
Patch-mainline: Merged into tipc-devel
References: bnc#797455

Make this rename so that it is consistent with the majority
of the other tipc structs and to assist in removing any
ambiguity with other similar names in other subsystems.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
Signed-off-by: Erik Hugne <erik.hugne@ericsson.com>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>

---
 net/tipc/bcast.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/net/tipc/bcast.c b/net/tipc/bcast.c
index 653be79..e88da1c 100644
--- a/net/tipc/bcast.c
+++ b/net/tipc/bcast.c
@@ -84,7 +84,7 @@ struct bcbearer {
 };
 
 /**
- * struct bclink - link used for broadcast messages
+ * struct tipc_bclink - link used for broadcast messages
  * @link: (non-standard) broadcast link structure
  * @node: (non-standard) node structure representing b'cast link's peer node
  * @bcast_nodes: map of broadcast-capable nodes
@@ -93,7 +93,7 @@ struct bcbearer {
  * Handles sequence numbering, fragmentation, bundling, etc.
  */
 
-struct bclink {
+struct tipc_bclink {
 	struct link link;
 	struct tipc_node node;
 	struct tipc_node_map bcast_nodes;
@@ -101,10 +101,10 @@ struct bclink {
 };
 
 static struct bcbearer bcast_bearer;
-static struct bclink bcast_link;
+static struct tipc_bclink bcast_link;
 
 static struct bcbearer *bcbearer = &bcast_bearer;
-static struct bclink *bclink = &bcast_link;
+static struct tipc_bclink *bclink = &bcast_link;
 static struct link *bcl = &bcast_link.link;
 
 static DEFINE_SPINLOCK(bc_lock);
-- 
1.7.8.3

